EnrichmentFormSample/EnrichmentFormSample/Scripts/blades/edit-address.tpl.html

```diff
@@ -0,0 +1,56 @@
+﻿<div class="blade-static __bottom">
+    <button class="btn" ng-click="bladeClose()">
+        <span>{{ 'platform.commands.cancel' | translate }}</span>
+    </button>
+    <button class="btn" ng-click="saveChanges()" ng-disabled="!isValid">
+        <span>{{ 'platform.commands.ok' | translate }}</span>
+    </button>
+</div>

```

Comment:
refactor to include ok-cancel.tpl.html from platform (1 line instead 8)

Reply:
I thought to do that but in this case we need to reference Catalog module. The problem is what version of module should we reference?

Reply:
why catalog? Platform is our framework

Reply:
I found the initial template in Catalog. Is there template in Platform and should Catalog use the same template ?

Reply:
repeating: "refactor to include ok-cancel.tpl.html from platform". And don't change anything in catalog module for now.